src/common/mfu_flist_remove.c

```diff
@@ -588,6 +589,81 @@ void mfu_flist_unlink(mfu_flist flist)
     int levels, minlevel;
     mfu_flist* lists;
     mfu_flist_array_by_depth(flist, &levels, &minlevel, &lists);
+    mfu_flist pstatlist;
+    uint64_t size = mfu_flist_size(flist);
+    char* strings[size];
```

Comment:
I don't know that this line is safe with all C compilers, so let's use a MFU_MALLOC here and a corresponding free later.